src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,204 @@
+"use client";
+
+import React from "react";
+import { Svg, Document, Page, Text, View, StyleSheet, Path } from "@react-pdf/renderer";
+import { faFlag, faSquare, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+
+import { ParcelOfSpecificDateAndLocation } from "@/components/DayOverview/DayOverview";
+
+interface DayOverviewDateProps {
+    date: Date;
+}
+
+interface DayOverviewLocationProps {
+    location: string;
+}
+
+interface DayOverviewRowProps {
+    row: ParcelOfSpecificDateAndLocation;
+}
+
+interface DayOverviewPdfProps {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateAndLocation[];
+}
+
+interface DayOverviewCardProps {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateAndLocation[];
+}
+
+interface CustomSVGProps {
+    icon: IconDefinition;
+    color: string;
+    fill: boolean;
+}
+
+const styles = StyleSheet.create({
+    page: {
+        display: "flex",
+        flexDirection: "column",
+        fontSize: "11pt",
+    },
+    card: {
+        margin: "0 30pt 15pt",
+    },
+    title: {
+        fontFamily: "Helvetica-Bold",
+        textTransform: "uppercase",
+        fontSize: "30px",
+        textAlign: "center",
+        margin: "0 22.5pt 22.5pt",
+    },
+    dateLocationWrap: {
+        justifyContent: "space-between",
+        marginBottom: "4pt",
+    },
+    svg: {
+        margin: "0 0.75pt",
+        padding: "0 0.75pt",
+    },
+    cell: {
+        borderStyle: "solid",
+        borderRightWidth: 1,
+        borderBottomWidth: 1,
+        padding: "2.25pt",
+    },
+    margin: { height: "37.5pt" },
+    row: { display: "flex", flexDirection: "row" },
```

Comment:
flex-direction row is default

Reply:
I tested and I think column is default

Reply:
I mean idk if there's some inheritance somehow due to React-pdf, but row is definitely default.

Reply:
Not sure. But if everything breaks below because I remove the line `flexDirection: row` then it means it has to be kept 
Maybe because the document is set at column (?)

Reply:
for some reason i have the same "issue" of docs saying row is default but whenever i try it it always seems like column is default TT